packages/paypal-js/src/load-script.ts

```diff
@@ -14,11 +14,17 @@ export function loadScript(
     PromisePonyfill: PromiseConstructor = Promise
 ): Promise<PayPalNamespace | null> {
     validateArguments(options, PromisePonyfill);
-
     // resolve with null when running in Node or Deno
     if (typeof document === "undefined") return PromisePonyfill.resolve(null);
 
+    if (!options.clientId) {
```

Comment:
Do we want to move this into `loadCustomScript` with the other validations? Seems like this might start breaking on incorrectly-implemented sites where another namespace exists and is used below.

Reply:
I originally had it there but I realized that (per the README)`loadCustomScript` is just a generic script loader that should work with any URL. Whereas `loadScript` is specifically for the JS SDK script.

Reply:
Good catch. Maybe to line 39 then? At least give
```js
// resolve with the existing global paypal namespace when a script with the same params already exists
    if (findScript(url, attributes) && existingWindowNamespace) {
        return PromisePonyfill.resolve(existingWindowNamespace);
 }```
 a chance to gracefully handle a bad implementation in production today?

Reply:
Oh actually, that wouldn't find the existing instance, because the URL would be different due to missing `clientId`.